src/program/snabb_lwaftr/setup.lua

```diff
@@ -22,10 +23,26 @@ function lwaftr_app(c, conf)
    config.app(c, "fragmenterv6", ipv6_apps.Fragmenter,
               { mtu=conf.ipv6_mtu })
 
-   config.link(c, "reassemblerv4.output -> lwaftr.v4")
```

Comment:
This is getting horribly non-compositional. I've made changes to the same code for NDP resolution. Any thoughts on keeping the complexity under control?